libkbfs/tlf_handle_resolve.go

```diff
@@ -195,28 +236,32 @@ type resolvableID struct {
 	id  keybase1.UserOrTeamID
 }
 
-func (ruid resolvableID) resolve(ctx context.Context) (nameIDPair, keybase1.SocialAssertion, error) {
+func (ruid resolvableID) resolve(ctx context.Context) (
+	nameIDPair, keybase1.SocialAssertion, tlf.ID, error) {
 	name, err := ruid.nug.GetNormalizedUsername(ctx, ruid.id)
 	if err != nil {
-		return nameIDPair{}, keybase1.SocialAssertion{}, err
+		return nameIDPair{}, keybase1.SocialAssertion{}, tlf.NullID, err
 	}
+	// TODO(KBFS-2621): get the TLF ID and display name for implicit
+	// teams. Blocked on CORE-6623.
```

Comment:
In that case, why don't we just call the to-be-added CreateTLF RPC into service here so that at the time the handle is returned it's guaranteed to have a TLF ID?

Reply:
@songgao: I don't understand your comment.  `resolvableID` is used when we start with a iteam ID (say, due to an incoming MD server notification, and we are reconstructing the handle from the MD) and need to resolve it backwards into a handle.  Can you clarify what that has to do with creating a TLF ID?

(It's possible this path won't ever be used for implicit teams, since the one source of incoming notifications without pre-registration I can think of -- rekeys -- won't be relevant.)

Reply:
whoops I put the comment in the wrong place when I was moving from my local git here after I got Internet. I'll add the comment in the right place. Sorry!